src/catalog/lib.rs

```diff
@@ -87,7 +87,7 @@ pub enum SchemaType {
 
 #[derive(Clone, Debug)]
 pub struct CatalogEntry {
-    inner: CatalogItem,
+    pub inner: CatalogItem,
```

Comment:
This shouldn't be `pub`. There's an accessor method `item()` you can use. (Feel free to rename this field to `item` if you'd like; I assume that's part of the reason you didn't find the accessor method.)